frontend/src/citizen-frontend/calendar/AbsenceModal.tsx

```diff
@@ -103,6 +104,8 @@ export default React.memo(function AbsenceModal({
   const i18n = useTranslation()
   const [lang] = useLang()
 
+  const holidayPeriods = useQueryResult(holidayPeriodsQuery())
+
```

Comment:
`holidayPeriods` is already available in the parent component. If passed from there, the `isSuccess` logic could also be skipped. This applies to all the 3 components changed in this PR.

Reply:
ah, good catch